### PR TITLE
Filter attendance by Minister or all MPs

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer

Could you please have a look at this before I send a pull-request upstream?

### CODE REVIEW

File: pombola/south_africa/views/attendance.py [hunk 1]
```diff
@@ -82,6 +86,21 @@ def get_context_data(self, **kwargs):
...
+                    ).overlapping_dates(
+                        annual_attendance['start_date'],
+                        annual_attendance['end_date'])
+
+                    active_minister_slugs = [am.person.slug for am in active_ministers]
+
+                    members_with_pa_url = [mbr_att for mbr_att in attendance_summary if
+                        mbr_att['member']['pa_url']]
+                    attendance_summary = [mbr_att for mbr_att in members_with_pa_url if
+                        mbr_att['member']['pa_url'].split('/')[-2] in active_minister_slugs]
```
Comment (xybrnet) [inline pombola/south_africa/views/attendance.py]: Is there a better way to do the 4 lines above?

**Comment by longhotsummer** [comment]
I don't have this setup locally -- could you include a screenshot?

**Comment by xybrnet** [comment]
mp-attendance-filter-by-position

mp-attendance-filtered

File: pombola/south_africa/views/attendance.py (hunk 1 above)
Comment (longhotsummer) [reply pombola/south_africa/views/attendance.py]: You could combine them:

```python
active_minister_slugs = set(am.person.slug for am in active_ministers)
attendance_summary = [m for m in attendance_summary
    if m['member']['pa_url'] and m['member']['pa_url'].split('/')[-2] in active_minister_slugs]
```

Apart from that, the slugs are the best way I know of right now to link the two datasets (our attendance and their MP information). But I don't know what the data in `annual_attendance` looks like.

Comment (xybrnet) [reply pombola/south_africa/views/attendance.py]: Thank you. That's simple enough.

`annual_attendance` is each of the items in the list returned here:
https://api.pmg.org.za/committee-meeting-attendance/summary/

**Comment by longhotsummer** [comment]
Looks great!

Gaile also asked for the banner wording to be changed and the font size reduced a bit.

> Please note: People's Assembly has suspended access to attendance of the MPs of small parties.

**Comment by longhotsummer** [comment]
Could we put the Ministers and deputy ministers first and make it the default?

It's more impactful because they're expected to attend.

**Comment by longhotsummer** [comment]
Looks good to me, thanks Damian.

**Comment by longhotsummer** [comment]
@xybrnet if you have a minute, please take another screenshot so I can send it to Gaile?

**Comment by xybrnet** [comment]
@longhotsummer 

screenshot at feb 08 09-24-42

**Pull request merged**